README.md

```diff
@@ -173,8 +173,8 @@ These are the current supported versions:
 - [x] 1.9
 - [x] 1.8
 - [x] 1.7
-- [x] 1.6
-- [x] 1.5
+- [ ] 1.6
+- [ ] 1.5
```

Comment:
Can we instead mark them as existing but experimental?

Reply:
Yup. Like this?